include/rocksdb/compaction_filter.h

```diff
@@ -32,6 +33,18 @@ struct CompactionFilterContext {
 
 class CompactionFilter {
  public:
+  enum ValueType {
```

Comment:
Most of that enum is internal stuff, and the values themselves are part of the file format. I don't think it belongs in public API.